utils/rpms/daos.spec

```diff
@@ -3,6 +3,13 @@
 %define agent_svc_name daos_agent.service
 %define sysctl_script_name 10-daos_server.conf
 
+%bcond_without server
+
+%if %{with server}
+%global daos_build_args FIRMWARE_MGMT=yes
```

Comment:
This is not a server-only feature. If it's enabled, it needs to be enabled on both client and server sides. I wouldn't be opposed to adding a new `bcond_without` for it, though.

Reply:
Are there server components needed for the client side?  I think we removed it because it failed to build